wasm-test-suite/wasm-suite-util_test.go

```diff
@@ -265,14 +264,20 @@ func mustUploadDir(dir, endpoint string) string {
 
 		absPath, err := filepath.Abs(fpath)
 		if err != nil {
-			panic(err)
+			return err
```

Comment:
AFAICT, the `must*` functions are expected to succeed or panic.

Reply:
It does already. The error is returned by `filepath.Walk()` and will trigger a panic some lines below.

But `panic(err)` may be easier to read, though. It probably would be best to change all occurences of `return err` into `panic(err)` inside of the `WalkFunc`. (Or leave them all at `return err`. Mixing both variants is a bit strange)